kafka-store/src/test/java/io/confluent/kafka/schemaregistry/storage/StringSerializer.java

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
It seems this could move from core rather than being copied. Doesn't look like it is used anymore since all the tests it was used in have moved.